proposals/0005-strict-initializer-lists.md

```diff
@@ -33,11 +33,13 @@ In HLSL the following code is valid:
     A a[2];
     int c;
   };
+  {% raw %}
   B b = {{1, 1.2}, {2, 2.2}, 3};   // Array elements specified as members
   B b2 = {1, 2, 3, 4, 5};          // each field initialized separately
   B b3 = {{1, {2, 3}}, {4, 5}};    // Completely random grouping of arguments
   int4 i4 = {1,2,3,4};             // valid int4 in C-syntax
   B b4 = {i4, 5};                  // int4 implicitly expanded to 4 arguments
+  {% endraw %}

```

Comment:
Are these because of the double braces `{{`?  We could also insert a space between them, but it's good to know you can escape a section like this too!

Reply:
Looks like we have another problem in [0002-cxx-attributes](https://llvm-beanz.github.io/hlsl-specs/proposals/0002-cxx-attributes.html) as well.  That's troubling...

Reply:
Oof... yea, I'll fix 0002.

The horrible thing is that Jekyll pre-processes all the source files with Liquid and Liquid's syntax is silly conflictive with other actual programming languages.

Reply:
Is there any other way to solve this?  How do we capture the rules and make sure documents don't contain misinterpreted patterns?